frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -218,14 +401,82 @@ export interface RenderedMarkdownProps {
   overrideComponents?: Components
 
   /**
-   * Only allows italics, bold, strikethrough, and emojis in button/download button labels
+   * Only allows italics, bold, strikethrough, and emojis in button/download button labels, does not allow colored text
    */
   isButton?: boolean
 
   /**
    * Only allows italics, bold, strikethrough, emojis, links, and code in widget/expander/tab labels
    */
   isLabel?: boolean
+
+  /**
+   * Does not allow colored text
+   */
+  isDownloadButton?: boolean
```

Comment:
Why do we need a special flag for `isDownloadButton` & `isFormSubmitButton`? Aren't these just normal buttons when it comes to UI/UX aspects?